README.md

```diff
@@ -1,3 +1,146 @@
 # Work in progress
 
-This repo is not ready for consumption, and is under heavy development
\ No newline at end of file
+This repo is not ready for consumption, and is under heavy development. Right now the Rust core bindings work only for Apple Silicon systems.
```

Comment:
works for 4 architectures 
<img width="357" alt="image" src="https://github.com/user-attachments/assets/e230a185-0bc0-4579-aa82-3624e8e06f70">

https://github.com/TBD54566975/web5-rs/blob/main/.github/workflows/bound-ci-kt.yml